app/views/handlers/confirm_email.py

```diff
@@ -65,8 +72,9 @@ def form(self):
         return generate_form(
             schema=self._schema,
             question_schema=self.question_schema,
-            answer_store=None,
-            metadata=None,
+            answer_store=self._questionnaire_store.answer_store,
+            list_store=self._questionnaire_store.list_store,
+            metadata=self._questionnaire_store.metadata,
```

Comment:
We should consider passing questionnaire_store instead of it's individual stores here.

Reply:
True, but this is done all over, IR, question, confirm_email, feedback, even block and submit could do it as well

Reply:
I mentioned it because list store is being added. I agree we should be consistent, and a card should be raised for that elsewhere in the codebase, but as this is being changed now I think it makes sense to rationalise to using `questionnaire_store` rather than adding `list_store`.